libs/dmlf/src/colearn/muddle_learner_networker_impl.cpp

```diff
@@ -223,6 +240,11 @@ void MuddleLearnerNetworkerImpl::PushUpdateBytes(UpdateType const &type_name, By
     auto next_ones = alg_->GetNextOutputs();
 
     Peers peers;
+    if (next_ones.empty())
+    {
+      FETCH_LOG_WARN(LOGGING_NAME, "PushUpdateBytes(2) got no peers from alg to send to.");
```

Comment:
What is (2)?

Shouldn't it be "... **has** no peers...?

Reply:
(2) is the second message of that form. "All error messages should be unique in the source"